templates/single-product/add-to-cart/variable.php

```diff
@@ -29,7 +29,7 @@
 	<?php do_action( 'woocommerce_before_variations_form' ); ?>
 
 	<?php if ( empty( $available_variations ) && false !== $available_variations ) : ?>
-		<p class="stock out-of-stock"><?php esc_html_e( 'This product is currently out of stock and unavailable.', 'woocommerce' ); ?></p>
+        <p class="stock out-of-stock"><?php echo apply_filters( 'woocommerce_out_of_stock_message', esc_html__( 'This product is currently out of stock and unavailable.', 'woocommerce' ) ); ?></p><?php // @codingStandardsIgnoreLine ?>
```

Comment:
`@codingStandardsIgnoreLine` is deprecated and we should avoid it, if necessary there we can ignore specific rules with `phpcs:ignore <rule>`.

Reply:
Hello @claudiosanches 

Thanks, I have removed the `@codingStandardsIgnoreLine` code. 

Please review and let me know.